index.html

```diff
@@ -2663,8 +2807,192 @@ <h3>#xmlLang-root</h3>
     </section> <!-- extensions-->
 
   <section class="appendix">
-      <h2>Acknowledgments</h2>
-      The editors would like to thank XXX for their contributions to this specification.
+    <h2>Registry Section</h2>
+    <section id="registry-definition">
+      <h2>Registry Definition</h2>
+      <p>This section specifies the <a>registry definition</a>, consisting of
+        the custodianship, change process and
+        the core requirements of the <a>registry tables</a> defined in this document.
+      </p>
+      <section id="custodianship">
+        <h3>Custodianship</h3>
+        <p>The <dfn data-cite="w3c-process#custodian">custodian</dfn> of this <a>w3c registry</a> is the
+          <dfn data-abbr="TTWG"><a href="https://www.w3.org/groups/wg/timed-text">Timed Text Working Group</a></dfn>. 
+          If the <a>TTWG</a> is unable to fulfil the role of <a>custodian</a>,
+          for example if it has been <a data-cite="w3c-process#GeneralTermination">closed</a>,
+          the <a>custodian</a> in lieu is the <a>W3C Team</a>.
+        </p>
+      </section>
+      <section id="change-process">
+        <h3>Change Process</h3>
+        <section id="requesting-a-change">
+          <h4>Requesting a change</h4>
+          <p>Changes to this  <a>W3C Registry</a> MUST be requested (the <dfn>change request</dfn>)
+            using any one of the following options:</p>
+          <ul>
+            <li>Open an issue on this document's <a class="pp-vcs" href="">version control system</a>;</li>
+            <li>Send an email to the <a>TTWG</a>'s public email reflector
+              <a href="mailto:public-tt@w3.org">public-tt@w3.org</a></li>
+          </ul>
+          <p>The <a>change request</a> MUST include enough information for the <a>custodian</a>
+            to be able to identify all of:</p>
+          <ul>
+            <li>The <a>registry table</a>;</li>
+            <li>The <a>registry entries</a> being proposed for addition;</li>
+            <li>The <a>provisional</a> <a>registry entries</a> for which changes are being proposed;</li>
+            <li>The <a>registry entries</a> being proposed for a <a>status</a> change;</li>
```

Comment:
quite minor, but above three items might be better to be quoted as `or`?

Reply:
I don't think so. I see this as three distinct lists, any of which could be empty.

If the lists are:
1. entries proposed for addition: `[foo]`
2. provisional entries being changed: `[bar]`
3. registry entries being proposed for a status change: `[]`

then it would not be acceptable to include only 1. or 2. but somehow obfuscate or hide the other.

Reply:
I agree if this can accept empty input. My concern was combination of `MUST` clause and `identify all of`.
> The <a>change request</a> MUST include enough information for the <a>custodian</a> to be able to identify all of:

(yeah, identify should allow identified as empty..)

Reply:
I could change "The ... entries" to "Any ... entries" or "All ... entries" if that would make it even clearer?

Reply:
Clarified in ef4ac7e as "All ... entries ..., if any" for each of those three affected bullets.

Reply:
+1 on this! thank you.